bragg-edge-imaging-2D.ipynb

```diff
@@ -329,7 +329,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sc.plot.plot(sc.sum(ds['sample'], dim='spectrum'))"
+    "# sc.plot.plot(sc.sum(ds['sample'], dim='spectrum'))"
```

Comment:
Is this supposed to still be commented?

Reply:
Yes, please keep for now. there is potential problem with scipp plotting I need to investigate.